roles/dcos_cli/templates/logstash.json.j2

```diff
@@ -0,0 +1,46 @@
+{
+  "id": "logstash",
+  "cmd": "/bin/bash -c \"/opt/logstash/bin/plugin install logstash-input-salesforce; logstash -f /config-dir/logstash.conf\"",
+  "instances": {{ dcos_cli_app_logstash_instances }},
+  "cpus": {{ dcos_cli_app_logstash_cpus }},
+  "mem": {{ dcos_cli_app_logstash_mem }},
+  "labels": {
+    "DCOS_PACKAGE_NAME": "logstash",
+    "DCOS_PACKAGE_IS_FRAMEWORK": "true"
```

Comment:
well even though its not using a dcos framework, for purposes of labelling its a framework for the platform, and not an app.